client/post-editor/editor-drawer/index.jsx

```diff
@@ -246,7 +247,10 @@ const EditorDrawer = React.createClass( {
 
 		const { plan } = this.props.site;
 		const hasBusinessPlan = isBusiness( plan ) || isEnterprise( plan );
-		if ( ! hasBusinessPlan ) {
+		const { privacySetting = 1 } = this.props;
```

Comment:
It doesn't appear that the default value here would ever be used; partly because the condition above `! this.props.privacySetting` would abort on a falsey value, but also because `mapStateToProps` always provides at least a `null` value based on the behavior of `getSiteSetting`. This would not trigger the default ([see example](http://jsbin.com/pecomacahu/edit?js,console)).